src/app/client/src/app/modules/dashboard/components/course-progress/course-progress.component.ts

```diff
@@ -419,6 +432,47 @@ export class CourseProgressComponent implements OnInit, OnDestroy, AfterViewInit
         this.populateCourseDashboardData();
       });
   }
+  getReportUpdatedOnDate() {
+    const batchId = _.get(this.queryParams, 'batchIdentifier');
+    const reportParams = {
+      'course-progress-reports': `course-progress-reports/report-${batchId}.csv`,
+      'assessment-reports': `assessment-reports/report-${batchId}.csv`
+    };
+    const url = `${this.config.urlConFig['URLS'].COURSE.GET_REPORTS_METADATA}`;
+    const requestParams = {
+      params: {
+        fileNames: JSON.stringify(reportParams)
+      }
+    };
+    this.usageService.getData(url, requestParams).subscribe((response: any) => {
+      if (_.get(response, 'responseCode') === 'OK') {
+        const courseProgressReportData = _.get(response, `result.course-progress-reports.lastModified`);
+        const assessmentReportData = _.get(response, `result.assessment-reports.lastModified`);
+        this.progressReportUpdatedOn =  courseProgressReportData ? moment(courseProgressReportData).format('DD/MM/YYYY') : '';
```

Comment:
Use angular pipe to show the date format.

Reply:
Fixed